### PR TITLE
Prevent double loading of libs in various places

### PR DESCRIPTION
**Pull request by phl0**
Seems we are double loading various libs in various places... Removed (at least some of) them.

### CODE REVIEW

**Review by HB9HIL — APPROVED** [review]
clicked around and couldn't find any issues

File: application/views/dashboard/index.php [hunk 1]
```diff
@@ -31,7 +31,9 @@ function echo_table_col($row, $name) {
 		case 'WWFF':    echo '<td>' . ($row->COL_WWFF_REF) . '</td>'; break;
 		case 'POTA':    echo '<td>' . ($row->COL_POTA_REF) . '</td>'; break;
 		case 'Grid':
-			$ci->load->library('qra');
+			if(!$this->load->is_loaded('Qra')) {
```
Comment (HB9HIL) [inline application/views/dashboard/index.php:34]:
```suggestion
			if(!$ci->load->is_loaded('Qra')) {
```

File: application/views/view_log/partial/log_ajax.php [hunk 2]
```diff
@@ -32,8 +32,10 @@ function echo_table_col($row, $name) {
 		case 'WWFF':    echo '<td>' . ($row->COL_WWFF_REF ?? '') . '</td>'; break;
 		case 'POTA':    echo '<td>' . ($row->COL_POTA_REF ?? '') . '</td>'; break;
 		case 'Grid':
-            $ci->load->library('qra');
-            echo '<td>' . ($ci->qra->echoQrbCalcLink($row->station_gridsquare, $row->COL_VUCC_GRIDS, $row->COL_GRIDSQUARE)) . '</td>'; break;
+				if(!$this->load->is_loaded('Qra') {
+					$this->load->library('Qra');
```
Comment (HB9HIL) [inline application/views/view_log/partial/log_ajax.php:36]:
```suggestion
				if(!$ci->load->is_loaded('Qra') {
					$ci->load->library('Qra');
```

**Review by HB9HIL — CHANGES_REQUESTED** [review]

File: application/views/view_log/partial/log_ajax.php (hunk 2 above)
Comment (phl0) [reply application/views/view_log/partial/log_ajax.php:36]: And even a closing parenthesis missing :)

**Review by HB9HIL — APPROVED** [review]

**Review by AndreasK79 — APPROVED** [review]

**Review by int2001 — APPROVED** [review]
less overhead.
tested varoius places which are affected by this PR. can't spot an error

**Pull request merged**